apps/code-examples/src/app/code-examples/flyout/flyout/basic/demo.component.ts

```diff
@@ -35,7 +35,7 @@ export class DemoComponent {
     });
   }
 
-  protected openSimpleFlyout(): void {
+  public openSimpleFlyout(): void {
```

Comment:
we are not using this in the unit test, can we leave this as protected.

Reply:
reverted